src/chrome/content/rules/ThisAirbnbDoesNotExist.com.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Mismatch:
+		www.thisairbnbdoesnotexist.com
```

Comment:
works for me, but the whole domain seems just an alias for thisrentaldoesnotexist.com

Reply:
@gloomy-ghost thanks, seems like they just fixed it, added!